<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistroEquipo</class>
 <widget class="QWidget" name="RegistroEquipo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>641</width>
     <height>521</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka Display&quot;;</string>
    </property>
    <property name="text">
     <string>Direcci√≥n ip:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka Display&quot;;</string>
    </property>
    <property name="text">
     <string>Numero de Serie:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka Display&quot;;</string>
    </property>
    <property name="text">
     <string>Prioridad:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka Display&quot;;</string>
    </property>
    <property name="text">
     <string>Precio de compra:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_NumSerie">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_IP">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_precio">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Priorid">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Alta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Media</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Baja</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka Display&quot;;</string>
    </property>
    <property name="text">
     <string>Tipo de Dispositivo:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_TipDisp">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Computador</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Impresora</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka Display&quot;;</string>
    </property>
    <property name="text">
     <string>Marca:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka Display&quot;;</string>
    </property>
    <property name="text">
     <string>Modelo:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Marca">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Modelo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka Display&quot;;</string>
    </property>
    <property name="text">
     <string>Observaciones:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_observ">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>311</width>
      <height>141</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Guardar">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>600</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>GUARDAR</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>60</y>
     <width>401</width>
     <height>191</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka Display&quot;;</string>
    </property>
    <property name="text">
     <string>Fecha de compra:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka Display&quot;;</string>
    </property>
    <property name="text">
     <string>Fecha fin garant√≠a:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka Display&quot;;</string>
    </property>
    <property name="text">
     <string>Fecha de instalaci√≥n:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_FechIns">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_FechFinGa">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_FechComp">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Sitka Display&quot;;</string>
   </property>
   <property name="text">
    <string>INVENTARIOS - CONTROL DE EQUIPOS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>680</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>SALIR</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget_CaracPc">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>250</y>
     <width>401</width>
     <height>191</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Computador</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit_Procesador">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Sitka Display&quot;;</string>
     </property>
     <property name="text">
      <string>Procesador:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Sitka Display&quot;;</string>
     </property>
     <property name="text">
      <string>Ram:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_Ram">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>16</number>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_So">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>110</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Sitka Display&quot;;</string>
     </property>
     <property name="text">
      <string>Sistema Operativo:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>641</width>
     <height>191</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);</string>
   </property>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka Display&quot;;</string>
    </property>
    <property name="text">
     <string>Departamento:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Depar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Empresa">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Contacto">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka Display&quot;;</string>
    </property>
    <property name="text">
     <string>Empresa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka Display&quot;;</string>
    </property>
    <property name="text">
     <string>Conctacto</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-17</y>
      <width>118</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget_ConfigTip">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>460</y>
     <width>401</width>
     <height>121</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Configuraci√≥n Tipo</string>
    </attribute>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Sitka Display&quot;;</string>
     </property>
     <property name="text">
      <string>Configuraciones Tipo:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_ConfigTip">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CargarConFt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Cargar Configuraci√≥n Tipo</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-30</y>
     <width>2371</width>
     <height>1231</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Fondos/Recursos/fondo13.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <action name="actionConfiguracion_Tipo">
   <property name="icon">
    <iconset resource="decorativos.qrc">
     <normaloff>:/Iconos/Recursos/iconoActualizar.png</normaloff>:/Iconos/Recursos/iconoActualizar.png</iconset>
   </property>
   <property name="text">
    <string>Configuracion Tipo</string>
   </property>
  </action>
  <action name="actionRealizar_Accion">
   <property name="icon">
    <iconset>
     <normaloff>:/Fondos/Recursos/martillo.png</normaloff>:/Fondos/Recursos/martillo.png</iconset>
   </property>
   <property name="text">
    <string>Realizar Accion</string>
   </property>
  </action>
  <zorder>label</zorder>
  <zorder>widget_2</zorder>
  <zorder>pushButton_Guardar</zorder>
  <zorder>widget</zorder>
  <zorder>label_12</zorder>
  <zorder>pushButton</zorder>
  <zorder>tabWidget_CaracPc</zorder>
  <zorder>widget_3</zorder>
  <zorder>tabWidget_ConfigTip</zorder>
 </widget>
 <resources>
  <include location="decorativos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>RegistroEquipo</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>790</y>
    </hint>
    <hint type="destinationlabel">
     <x>931</x>
     <y>813</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
